airbyte-integrations/connectors/source-ebay-finance/README.md

```diff
@@ -0,0 +1,33 @@
+# Ebay Finance
+This directory contains the manifest-only connector for `source-ebay-finance`.
```

Comment:
PR fix suggested here https://github.com/airbytehq/airbyte/pull/56920#pullrequestreview-2730204430:
 
 Renamed `source-ebay` to `source-ebay-finance`